cypress/integration/articleFlows/pinArticle.spec.js

```diff
@@ -169,6 +169,8 @@ describe('Pin an article - Admin User', () => {
       cy.findAllByRole('link', { name: 'Manage' }).first().click();
     });
 
+    cy.findByRole('heading', { name: 'Tools:' });
```

Comment:
We also change page on lines 179, 184, and 200 - I think to future proof we should add similar guards at each page change, otherwise the same issue might just pop up again later in the test run

Reply:
Will do here, once and for all 🤣